gbpservice/nfp/config/mode_shift.conf

```diff
@@ -0,0 +1,24 @@
+#------------------------#
+# MODE SHIFT INSTRUCTION #
+#------------------------#
+# NOTE: The present scope of the script supports only ADVANCED to ENTERPRISE migration.
+#       Please see that the machine has atleast 16 GB RAM left.
+
+FROM=advanced
+TO=enterprise
+
+#----------------------------------#
+# enterprise mode specific options #
+#----------------------------------#
+# Visibility GIT path
+# Default visibility repo https://github.com/oneconvergence/visibility.git
+# NOTE: Mandatory parameters
+
+VISIBILITY_GIT_BRANCH=master
+GIT_ACCESS_USERNAME=
```

Comment:
Devstack installation is currently dependent on git. We are getting code from GIT. Infact, devstack will get the code from GIT.